apps/web/src/pages/Teams/TeamMembersPage/TeamMembersPage.tsx

```diff
@@ -14,12 +88,114 @@ const TeamMembersPage = () => {
     description: "Page title for the view team members page",
   });
 
+  const columns = React.useMemo<ColumnsOf<TeamMember>>(
+    () => [
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Actions",
+          id: "OxeGLu",
+          description: "Title displayed for the team table actions column",
+        }),
+        accessor: (d) => `Actions ${d.id}`,
+        Cell: ({ row: { original: member } }: TeamMemberCell) =>
+          actionCell(member, team, roles),
+        disableGlobalFilter: true,
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Name",
+          id: "AUOq9D",
+          description:
+            "Title displayed for the team members table Name column.",
+        }),
+        accessor: (d) => getFullNameLabel(d.firstName, d.lastName, intl),
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Email",
+          id: "3/lHSy",
+          description:
+            "Title displayed for the team members table email column.",
+        }),
+        accessor: (d) => d.email,
+        Cell: ({ value }: TeamMemberCell) => emailLinkCell(value, intl),
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Membership roles",
+          id: "4Washm",
+          description:
+            "Title displayed for the team members table roles column.",
+        }),
+        accessor: (d) => d.id, // TO DO: Update with real role shape
```

Comment:
Probably should be role name for sorting, right?

Reply:
Are you thinking comma separate list of alphanumerically sorted roles? I know it is only one right now but there is nothing about the data model explicitly restricting multiples right now.

Reply:
Oh, forgot it was a collection.  Yeah, String.join a list of alpha sorted role names makes sense to me.

Reply:
I seeded an additional team-based role to play around with things

![image](https://user-images.githubusercontent.com/40485260/223807787-b9715f8f-c6ed-435c-9127-ab1bc7f7929d.png)

playing around with it

https://recordit.co/lKPwcmRTzs

for the most part, seems to work with a few quirks

- when a member has one role added, with one more available, drop down looks fine, but when they have two roles, drop down shows two roles to add
- can't remove a single role from a member it seems?

Reply:
looking at the issue, removing one role from a member isn't in scope

Reply:
Yeah, the remove dialog is to remove them from the team rather than remove roles. Maybe edit should use `sync` instead of `attach` 🤔 What do you think @petertgiles

Reply:
Yeah, sync makes sense to me since the edit dialog represents the desired state, not just a diff.

Reply:
Sorted list done in [update role accessor to be alpha sorted csv](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5811/commits/ad4d069c3bba1ec7852fe6ad45a0a8c91ad7821b)

And, the add -> sync change done in [update edit dialog to sync roles instead of attach](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5811/commits/b00e6f1ff58c4d8b865843dbe121e6819e0afb71)